src/adaptation/ecap/Registry.h

```diff
@@ -8,3 +8,7 @@
 
 // TBD
 
+#ifndef SQUID_SRC_ADAPTATION_ECAP_REGISTRY_H
+#define SQUID_SRC_ADAPTATION_ECAP_REGISTRY_H
```

Comment:
```suggestion
#ifndef SQUID_SRC_ADAPTATION_ECAP_REGISTRY_H
#define SQUID_SRC_ADAPTATION_ECAP_REGISTRY_H

// TBD
```

or just drop this unused file entirely.